pages/cart.html

```diff
@@ -0,0 +1,164 @@
+                    ><span class="material-icons">remove</span></a
+                  >
+                </div>
+                <span>₹299</span><span class="p-l-1"><strike>₹399</strike></span>
+              </div>
+            </div>
+
+            <button class="btn btn-outline-warning m-t-1 m-b-1">
+              Move to wishlist
+            </button>
+            <button class="btn btn-outline-danger">Remove from cart</button>
+          </div>
+
+          <div class="card card-shadow p-1">
+            <div class="flex-row">
+              <div class="p-1 image">
+                <img src="/assets/top picks.png" alt="Biryani" />
+              </div>
+              <div>
+                <div class="card-heading-container">
+                  <h3 class="font-md font-medium">Behrouz</h3>
+                  <h5 class="font-sm font-light p-b-1">Biryani</h5>
+                </div>
+                <span>Quantity: </span>
+                <div class="card-text-container flex-center">
+                  <a href="#"> <span class="material-icons">add</span> </a>
+                  <input class="slider-text-input" type="text" value="2" />
+                  <a href="#"> <span class="material-icons">remove</span></a>
+                </div>
+                <span>₹299</span><span class="p-l-1"><strike>₹399</strike></span>
+              </div>
+            </div>
+
+            <button class="btn btn-outline-warning m-t-1 m-b-1">
+              Move to wishlist
+            </button>
+            <button class="btn btn-outline-danger">Remove from cart</button>
+          </div>
+
+          <div class="card card-shadow p-1">
+            <div class="flex-row">
+              <div class="p-1 image">
+                <img src="/assets/premium-category.png" alt="burger" />
+              </div>
+              <div>
+                <div class="card-heading-container">
+                  <h3 class="font-md font-medium">Mcdonald's</h3>
+                  <h5 class="font-sm font-light p-b-1">Burger</h5>
+                </div>
+                <span>Quantity: </span>
+                <div class="card-text-container flex-center">
+                  <a href="#"><span class="material-icons">add</span></a>
+                  <input class="slider-text-input" type="text" value="1" />
+                  <a href="#"><span class="material-icons">remove</span></a>
+                </div>
+                <span>₹299</span><span class="p-l-1"><strike>₹399</strike></span>
+              </div>
+            </div>
+
+            <button class="btn btn-outline-warning m-t-1 m-b-1">
+              Move to wishlist
+            </button>
+            <button class="btn btn-outline-danger">Remove from cart</button>
+          </div>
+        </div>
+
+        <div class="flex-center checkout-details">
+          <div class="p-1 card card-shadow">
+            <div class="font-sm font-bold p-1">Price details</div>
+            <hr>
+            <div class="p-l-1 p-b-1">
```

Comment:
Just a suggestion, try to make these names more understandable

Reply:
will make them more readable in refactor.